tutorials/image-curation/image-curation.ipynb

```diff
@@ -0,0 +1,528 @@
+    "\n",
+    "To download the dataset, we are going to use a tool called img2dataset. Let's install it and download the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install img2dataset"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need to get a list of URLs that identify where all the images are hosted"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!wget https://huggingface.co/datasets/ChristophSchuhmann/MS_COCO_2017_URL_TEXT/resolve/main/mscoco.parquet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We truncate this list of URLs so we don't download the whole dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "\n",
+    "NUM_URLS = 20_000\n",
+    "urls = pd.read_parquet(\"mscoco.parquet\")\n",
+    "truncated_urls = urls[:NUM_URLS]\n",
+    "truncated_urls.to_parquet(\"truncated_mscoco.parquet\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, let's start the download."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "plaintext"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!img2dataset \\\n",
+    "    --url_list truncated_mscoco.parquet \\\n",
+    "    --input_format \"parquet\" \\\n",
+    "    --output_folder mscoco \\\n",
+    "    --output_format webdataset \\\n",
+    "    --url_col \"URL\" \\\n",
+    "    --caption_col \"TEXT\" \\\n",
+    "    --processes_count 16 \\\n",
+    "    --thread_count 64 \\\n",
+    "    --resize_mode no"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "\n",
+    "os._exit(0) # Shut down all kernels to prevent dependencies from conflicting"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Install NeMo Curator"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
+   "outputs": [],
+   "source": [
+    "!pip install cython ipywidgets\n",
+    "# Install from source by default\n",
+    "!pip install --extra-index-url https://pypi.nvidia.com ../../[image]\n",
+    "# !pip install --extra-index-url https://pypi.nvidia.com nemo-curator[image]\n",
+    "%env DASK_DATAFRAME__QUERY_PLANNING False"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Create CLIP Embeddings\n",
```

Comment:
Maybe explain a little more about what this means, what we're doing, and what the goal is?